source/agora/crypto/Key.d

```diff
@@ -210,12 +252,16 @@ public struct PublicKey
             `boa1xrra39xpg5q9zwhsq6u7pw508z2let6dj8r5lr4q0d0nff240fvd27yme3h`;
         PublicKey pubkey = PublicKey.fromString(address);
 
-        () nothrow @safe @nogc
+        () nothrow @safe
         {
             auto hash = hashFull(pubkey);
-            auto exp_hash = Hash("0xdb3b785e31c05c9383f498aa12a5c054fcb6f99b2" ~
-                "0bf7ff25fdb24d5fe6d1bb5768f1821de9d5d31faa7ebafc79837ba32dc4" ~
-                "774bffea9918411d0c4c8ac403c");
+            auto exp_hash = Hash("0x39007e97ccdc88c27bb8bded39016d95dfee4713a" ~
+                "c08e9be6025a33e64179effa8b827a5115f2a570f0ac16e9fa9f3c4e51f4" ~
+                "6642a1c6b40a2141c86418aea11");
```

Comment:
Since the HRP can be freely modified, it shouldn't affect the hash, otherwise a node could just mount attacks by using different HRP for different nodes.

Reply:
Reverted